docs/related-projects.md

```diff
@@ -142,19 +142,12 @@ cd retirement
 If the satellite app needs any Python requirements that are not specified in 
 [the cfgov-refresh requirements](https://github.com/cfpb/cfgov-refresh/tree/master/requirements/), 
 they will need to be installed seperately by 
-[accessing the Python containers' shell](#access-the-containers-shell) 
+[accessing a Python container shell](#access-a-container-shell) 
```

Comment:
I suggested removing the link from this text earlier because it's an anchor link that doesn't point to a section on this page. Likely due to an errant copy-paste from `running-docker.md`. The link should either be removed as initially suggested or possibly updated to point to `../running-docker/#access-a-containers-shell`.

In either case, the text should be updated for clarity, since there is only a single Python container: "accessing the Python container's shell" instead of "accessing a Python container shell".